gauss2d/utils.py

```diff
@@ -0,0 +1,179 @@
+# This file is part of gauss2d.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+from _gauss2d import loglike_gaussians_pixel as loglike_gaussians_pixel_pb
+
+
+def covar_to_ellipse(sigma_x_sq, sigma_y_sq, cov_xy, degrees=False):
+    """ Convert covariance matrix terms to ellipse major axis, axis ratio and position angle representation.
```

Comment:
I know this is for `lsst-dm` repository, but I would still avoid starting the docstring with a space.